Demo/Common/Configuration/DropInSettingsView.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import AdyenCard
+import SwiftUI
+
+@available(iOS 13.0.0, *)
+internal struct DropInSettingsView: View {
+    @ObservedObject internal var viewModel: ConfigurationViewModel
+
+    internal var body: some View {
+
+        NavigationView {
+            List {
+                Section {
+                    Toggle(isOn: $viewModel.allowDisablingStoredPaymentMethods) {
+                        Text("Disable Stored Payment Methods")
+                    }
+
+                    Toggle(isOn: $viewModel.allowsSkippingPaymentList) {
+                        VStack(alignment: .leading) {
+                            Text("Skip Payment List")
+                        }
+                    }
+                    Toggle(isOn: $viewModel.allowPreselectedPaymentView) {
+                        Text("Allow Preselected PaymentView")
```

Comment:
Should I add this "Allow" text? If yes, then can I add it to all properties?

Reply:
Maybe "Skip 'OneClick' view for stored payment" ? 🤔

Reply:
imo "Skip 'OneClick' view for stored payment"  is confusing 🤔 🤔